### PR TITLE
Add mention of str.search.* variables in the help of iz and izz

### PR DESCRIPTION
**Pull request by w0rm29**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [x] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Mention of variable `str.search.*` in iz and izz commands

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: librz/core/cmd_descs/cmd_info.yaml [hunk 1]
```diff
@@ -437,7 +437,7 @@ commands:
     args: []
   - name: iz
     cname: cmd_info_strings
-    summary: List strings
+    summary: List strings(str.search.*)
```
Comment (wargio) [inline librz/core/cmd_descs/cmd_info.yaml]:
```suggestion
    summary: List strings (see also str.search.*)
```

Comment (wargio) [reply librz/core/cmd_descs/cmd_info.yaml]: same below.

Comment (w0rm29) [reply librz/core/cmd_descs/cmd_info.yaml]: done

**Review by XVilka — CHANGES_REQUESTED** [review]
I recommend putting them into the details of the command instead.

**Review by XVilka — CHANGES_REQUESTED** [review]
Completely wrong change that wouldn't even compile

**Comment by w0rm29** [comment]
> Completely wrong change that wouldn't even compile

where to find the details of the command?

**Comment by wargio** [comment]
you need to rebuild and commit. you cannot just push the changes on the yaml. be sure to have also pyyaml installed

**Comment by w0rm29** [comment]
> you need to rebuild and commit. you cannot just push the changes on the yaml. be sure to have also pyyaml installed

while rebuilding, I need to use the below command to rebuild

`meson setup --wipe`

File: librz/core/cmd_descs/cmd_info.yaml (hunk 1 above)
Comment (w0rm29) [reply librz/core/cmd_descs/cmd_info.yaml]: this is working when i am rebuilding

**Comment by w0rm29** [comment]
> I recommend putting them into the details of the command instead.

i am not able to find this, should I add a property detail?

**Comment by wargio** [comment]
nono. the yaml files are used to generate the updated sources and headers by meson if pyYAML is installed.

**Comment by wargio** [comment]
https://github.com/rizinorg/rizin/blob/dev/librz/core/cmd_descs/meson.build#L37

**Comment by w0rm29** [comment]
> https://github.com/rizinorg/rizin/blob/dev/librz/core/cmd_descs/meson.build#L37

okay
so, where to add the `str.search.*` ?
earlier where i added is working
can I go ahead with it @XVilka @wargio ?

**Comment by wargio** [comment]
I think what he meant is to do something like this: https://github.com/rizinorg/rizin/blob/dev/librz/core/cmd_descs/cmd_write.yaml#L92

File: librz/core/cmd_descs/cmd_info.yaml [hunk 2]
```diff
@@ -436,8 +436,9 @@ commands:
       - RZ_OUTPUT_MODE_JSON
     args: []
   - name: iz
-    cname: cmd_info_strings(see also str.search.*)
+    cname: cmd_info_strings
     summary: List strings
+    description: see also str.search.*
```
Comment (wargio) [inline librz/core/cmd_descs/cmd_info.yaml]: Be a bit more descriptive: `Lists the strings found in a binary; the behavior can be modified via str.search.*`

File: librz/core/cmd_descs/cmd_info.yaml [hunk 3]
```diff
@@ -447,8 +448,9 @@ commands:
       - RZ_OUTPUT_MODE_QUIETEST
     args: []
   - name: izz
-    cname: cmd_info_whole_strings(see also str.search.*)
+    cname: cmd_info_whole_strings
     summary: List strings in the whole binary
+    description: see also str.search.*
```
Comment (wargio) [inline librz/core/cmd_descs/cmd_info.yaml]: Same here: `Lists the strings of the whole binary; the behavior can be modified via str.search.*`

File: librz/core/cmd_descs/cmd_info.yaml [hunk 4]
```diff
@@ -438,7 +438,7 @@ commands:
   - name: iz
     cname: cmd_info_strings
     summary: List strings
-    description: see also str.search.*
+    description: Lists the strings of the whole binary; the behavior can be modified via str.search.*
```
Comment (wargio) [inline librz/core/cmd_descs/cmd_info.yaml]: check the comments. this doesn't show the strings of the whole binary

**Comment by wargio** [comment]
as explained above, please install PyYAML and rebuild, then commit the new changes from the generated files.

**Review by wargio — APPROVED** [review]

**Comment by w0rm29** [comment]
why did the check failed
is it because of my change
I did build and it was successful

@wargio

**Comment by wargio** [comment]
don't worry about it. looks ready.

**Pull request merged**